src/sc_scda.h

```diff
@@ -1045,7 +1068,15 @@ sc_scda_fcontext_t *sc_scda_fread_varray_data (sc_scda_fcontext_t * fc,
  *                              something else on invalid arguments.
  */
 int                 sc_scda_ferror_class (sc_scda_ferror_t errcode,
-                                          sc_scda_ferror_t *errclass);
+                                          sc_scda_ferror_t * errclass);
+
+/** Check if a scda_errorcode_t encodes success.
+ *
+ * \param [in]    errorcode     An errcode that is output by a sc_scda function.
+ * \return                      True if \b errcode encodes success and
+ *                              false otherwise.
+ */
+int                 sc_scda_is_success (sc_scda_ferror_t errorcode);
```

Comment:
Thanks! I'll be merging this one now.

For future development: Would it be more intuitive to rename the function to sc_scda_ferror_success (or something else -- open for suggestions) and to return MPI_SUCCESS on success and some MPI error otherwise?

Reply:
I think that a function `sc_scda_ferror_success` that returns MPI error codes may be unexpected to the user and we also have the `sc_scda_ret_t` value `SC_SCDA_FERR_SUCCESS` that may be named too similar.
Do you think it is sensible to name the function `sc_scda_ferror_is_success` to be consistent to the other scda function names but still use the semantic of mapping an error code to a Boolean?